### PR TITLE
sidebar updates

### PR DESCRIPTION
**Pull request by jacktrusler**
### Sidebar updates

- Removed background image to match figma
- changed sidebar links to match figma
- extracted a component for 2 bottom links
- changed 2 bottom links to point to other websites

### CODE REVIEW

File: src/modules/components/SidebarNavbar/SidebarNavbar.tsx [hunk 1]
```diff
@@ -32,7 +32,12 @@ const validateSidebar = [
   {
     name: 'Minipool Dashboard',
     href: '/dashboard/',
-    icon: <IoStatsChart color="#ffffff" size={24} />,
+    icon: <BiBarChartAlt color="#ffffff" size={24} />,
```
Comment (Namaskar-1F64F) [inline src/modules/components/SidebarNavbar/SidebarNavbar.tsx:35]: `#fff` to save a few bytes. `white` to be more readable

File: src/modules/components/SidebarNavbar/Sidebar/SidebarItem.tsx [hunk 2]
```diff
@@ -9,12 +9,14 @@ type Props = {
...
 
 export default function SidebarItem({ item, setSidebarOpen }: Props) {
+  const includesHttp = item.href.toLowerCase().includes('http')
   return (
     <NextLink
       className="flex w-full items-center text-base"
       href={item.href}
       key={item.name}
       onClick={setSidebarOpen ? () => setSidebarOpen(false) : undefined}
+      target={includesHttp ? '_blank' : ''}
```
Comment (Namaskar-1F64F) [inline src/modules/components/SidebarNavbar/Sidebar/SidebarItem.tsx:19]:
```suggestion
      target={includesHttp && '_blank'}
```

File: src/modules/components/SidebarNavbar/Sidebar/SidebarItem.tsx [hunk 3]
```diff
@@ -9,12 +9,14 @@ type Props = {
 }
 
 export default function SidebarItem({ item, setSidebarOpen }: Props) {
+  const includesHttp = item.href.toLowerCase().includes('http')
```
Comment (Namaskar-1F64F) [inline src/modules/components/SidebarNavbar/Sidebar/SidebarItem.tsx:12]:
```suggestion
  const isExternalLink = item.href.toLowerCase().includes('http')
```

**Review by Namaskar-1F64F — APPROVED** [review]
LGTM, left a few nits

File: src/modules/components/SidebarNavbar/Sidebar/SidebarItem.tsx (hunk 2 above)
Comment (jacktrusler) [reply src/modules/components/SidebarNavbar/Sidebar/SidebarItem.tsx:19]: You need to do the ternary to avoid a console error.

**Pull request merged**